### PR TITLE
Update backup and restore actuator endpoint port

### PR DESCRIPTION
**Pull request by mesellings**
## Description

This closes issue https://github.com/camunda/camunda-docs/issues/4487.

> Starting version 8.6, the actuator endpoints, including backup, were moved to port 9600.
> However, the documentation for 8.6 still mentions the old port, 8080: https://docs.camunda.io/docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup/

Updated documentation for Next and 8.6:
- Changed port 8080 to 9600 in page.
- Changed to a breaking changes warning and added as an item.

## When should this change go live?

- [X] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [X] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@camunda/tech-writers I'm wondering whether this should be in Next at all, as this was for 8.6, so theoretically now a breaking change in Next (8.7)?

**Comment by mesellings** [comment]
@mihail-ca There was already a note on this page about breaking changes, but I've changed it to a warning and added in the port entry as well as changing the port references in the page.

**Review by conceptualshark — COMMENTED** [review]
If this needs to be changed for all references of the actuator endpoint, there's a small number of other places in the docs where it is referenced - it's not unique to backup and restore.

File: docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md [hunk 1]
```diff
@@ -5,12 +5,13 @@ description: "How to perform a backup and restore of Operate and Tasklist data."
 keywords: ["backup", "backups"]
 ---
 
-:::note
+:::warning breaking changes!
```
Comment (conceptualshark) [inline docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md:8]: I think we can afford to remove this note's old changes, and only include the new one now. It's populated throughout every version of the docs, and I assume doesn't need to be quite so loud in /next or 8.6, and would draw better attention to the port change.

**Comment by mesellings** [comment]
@conceptualshark @mihail-ca Please re-review

File: docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md [hunk 2]
```diff
@@ -5,13 +5,8 @@ description: "How to perform a backup and restore of Operate and Tasklist data."
...
 
-:::note
-This release introduces breaking changes, including:
-
-- The [get backup state API and response codes](#get-backup-state-api).
-- The utilized URL has changed. For example, `curl 'http://localhost:8080/actuator/backups'` rather than the previously used `backup`.
-- `backupId` must be of integer type now instead of string, which is in sync with Zeebe `backupId` requirements.
-
+:::warning breaking changes!
+In this release, the actuator endpoints (including backup) changed to use port 9600 instead of 8080. You will need to update your applications to use port 9600.
```
Comment (conceptualshark) [inline docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md:9]: I don't know what we define as a breaking change - this doesn't disrupt running applications/won't fail an upgrade, but may need a configuration adjustment.

```suggestion
As of the Camunda 8.6 release, the `/actuator` endpoints (including `/backups`) now default to port 9600. Ensure your `management.server.port` configuration parameter is correctly set before continuing.
```

Comment (mesellings) [reply docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md:9]: @ThorbenLindhauer @mihail-ca Does this definitely constitute a breaking change, or is it more of a configuration issue for people to be aware of?

Comment (ThorbenLindhauer) [reply docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md:9]: Yes, I'd say so. Default behavior of the product is changing, so applications that build on this default behavior will break. In this case, for example automations that people have written for orchestrating the backup procedures.

**Comment by mesellings** [comment]
Thanks for the input @conceptualshark please re-review - @ThorbenLindhauer @mihail-ca Are you happy with the proposed change?

image

**Review by conceptualshark — APPROVED** [review]
My point re: breaking changes is that I can't tell if that then warrants an update to Announcements, if our default style is a caution or a warning, or if a breaking change warning like this would be better suited to a different page - the component config, for example. I think this is fine as-is so customers can be alerted, but I think the initial ticket warrants further follow up.

**Review by ThorbenLindhauer — APPROVED** [review]
Looks good to me. I agree we should mention this on https://docs.camunda.io/docs/next/reference/announcements/#camunda-86 too.

**Comment by mesellings** [comment]
@conceptualshark @ThorbenLindhauer I've created a new issue for the follow-up work on this: https://github.com/camunda/camunda-docs/issues/4494

**Pull request merged**